e2e/react-router/basic-file-based/tests/scrollIntoView.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { expect, test } from '@playwright/test'
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/anchor')
+})
+
+// Testing the `Link` component with the `hashChangeScrollIntoView` prop
+test('Navigating via anchor `Link` with default hash scrolling behavior', async ({
+  page,
+}) => {
+  await expect(
+    page.getByRole('heading', { name: 'Default Anchor' }),
```

Comment:
please use getByTestId instead of getByRole (this needs a `data-testId` prop on the element you want to find

Reply:
Updated, thanks.